src/utils/APIBreaks/3.0.0.ts

```diff
@@ -0,0 +1,63 @@
+import { APIBreakInfo } from "./APIBreak";
+
+export const breakInfos300: APIBreakInfo[] = [
+  {
+    brokenAPIs: [{ name: "ns.nFormat" }],
+    info:
+      "ns.nFormat() was removed.\n" +
+      "Use ns.formatNumber, ns.formatRam, ns.formatPercent, or JS built-in objects/functions (e.g., Intl.NumberFormat, " +
+      "Intl.PluralRules, Intl.Locale) instead.",
+    showPopUp: true,
+  },
+  {
+    brokenAPIs: [
+      {
+        name: "ns.getTimeSinceLastAug",
+        migration: {
+          searchValue: "ns.getTimeSinceLastAug()",
+          replaceValue: "(Date.now() - ns.getResetInfo().lastAugReset)",
+        },
+      },
+      {
+        name: "ns.getPlayer().playtimeSinceLastAug",
+        migration: {
+          searchValue: "ns.getPlayer().playtimeSinceLastAug",
+          replaceValue: "(Date.now() - ns.getResetInfo().lastAugReset)",
+        },
+      },
+      {
+        name: "ns.getPlayer().playtimeSinceLastBitnode",
+        migration: {
+          searchValue: "ns.getPlayer().playtimeSinceLastBitnode",
+          replaceValue: "(Date.now() - ns.getResetInfo().lastNodeReset)",
+        },
+      },
+      {
+        name: "ns.getPlayer().bitNodeN",
+        migration: {
+          searchValue: "ns.getPlayer().bitNodeN",
+          replaceValue: "ns.getResetInfo().currentNode",
+        },
+      },
+      {
+        name: "ns.corporation.getCorporation().state",
+        migration: {
+          searchValue: "ns.corporation.getCorporation().state",
+          replaceValue: "ns.corporation.getCorporation().nextState",
+        },
+      },
+    ],
+    info:
+      "ns.getTimeSinceLastAug was removed.\n" +
+      "Use Date.now() - ns.getResetInfo().lastAugReset instead.\n\n" +
```

Comment:
```suggestion
      "It has been automatically replaced with "Date.now() - ns.getResetInfo().lastAugReset".\n\n" +
```
Seeing the final file output made me realize these should be reworded. It's not a call to action, but rather a notification of something that already occurred, in case you want to review it.

Reply:
> Seeing the final file output made me realize these should be reworded. It's not a call to action, but rather a notification of something that already occurred, in case you want to review it.

Done.

> We might want to make the distinction between the two groups more clear in some other way too, although if that's annoying then this is fine.

I'm okay with doing that, but I have no idea what the change should be.